Templates/CSharp/Model/EnumType.cs.tt

```diff
@@ -35,6 +39,9 @@ namespace <#=enumT.Namespace.GetNamespaceName()#>
         /// <summary>
         /// <#=emumMember.Name.ToUpperFirstChar().SplitCamelCase()#>
         /// </summary>
+<# if (emumMember.Deprecation != null) { #>
+        <#=string.Format("[Obsolete(\"{0}\")]", emumMember.Deprecation.Description)#>
```

Comment:
what's the benefit of using a string format here? (x3) IMHO it makes reading the template harder.